README.md

```diff
@@ -11,34 +11,33 @@ ___
 
 <!-- toc -->
 - [What is Enceladus?](#what-is-enceladus)
-    - [Menas](#wie-menas)
-    - [Standardization](#wie-standardization)
-    - [Conformance](#wie-conformance)
-- [How to build](#build)
-- [How to run](#run)
-- [How to contribute](#contribute)
-- [How to use](#use)
-    - [Standardization](#use-standardization)
+    - [Menas](#menas)
+    - [Standardization](#standardization)
+    - [Conformance](#conformance)
+- [How to build](#how-to-build)
+- [How to run](#how-to-run)
+- [How to contribute](#how-to-contribute)
+- [Documentation](#documentation)
 <!-- tocstop -->
 
-## <a name="what-is-enceladus"/>What is Enceladus?
+## What is Enceladus?
```

Comment:
Why do this? I know some pre-processors will handle it, but some don't. I would rather be safe.

Reply:
I believe auto-anchoring of headers is the standard of markdown. And injected html is not encouraged.

Reply:
I think this is fine, it works and it looks good.